### DISCUSSION TITLE
有老哥知道现在23版本的immortalwrt用拨号上网的话能跑满千兆网吗？

### DISCUSSION THREAD

**Discussion by tiantangzhixia** [immortalwrt/immortalwrt#1647] [Q&A]
之前有看到网上issues说opwrt22及以后的版本用拨号上网的话跑不满千兆网，好像只能500~600M左右的样子。现在23版本的immortalwrt有人用来拨号上网吗？能否跑满千兆？arm架构或者x86架构的有人测试过吗？

**Comment by toss-a** [comment]
![QQ20241231-142730](https://github.com/user-attachments/assets/002fd4ea-2c19-465e-b491-f23fa53fa5dd)
CPU AMD5825U 分配8个核心
